frontend/public/components/operator-lifecycle-manager/descriptors/status/index.tsx

```diff
@@ -8,30 +8,44 @@ import { ResourceLink } from '../../../utils';
 import { StatusCapability, CapabilityProps, DescriptorProps } from '../types';
 import { Phase } from './phase';
 import { PodStatusChart } from './pods';
+import { Tooltip } from '../../../utils/tooltip';
+import { Conditions } from '../../../conditions';
 
-// TODO(alecmerdler): Move these to separate module?
-const Default: React.SFC<StatusCapabilityProps> = ({value}) => _.isEmpty(value) && !_.isNumber(value)
-  ? <span className="text-muted">None</span>
-  : <span>{value}</span>;
+const Invalid: React.SFC<StatusCapabilityProps> = (props) => <span className="text-muted">
+  <i className="fa fa-exclamation-triangle text-warning" aria-hidden="true" />&nbsp;&nbsp;The field <code>status.{props.descriptor.path}</code> is invalid
+</span>;
+
+const Default: React.SFC<StatusCapabilityProps> = ({value}) => {
+  return _.isEmpty(value) && !_.isNumber(value) && <span className="text-muted">None</span>
+    || _.isObject(value) && <div style={{display: 'flex', flexDirection: 'column'}}>{_.map(value, (v, k) => <span key={k}>{k}: {v}</span>)}</div>
+    || <span>{_.toString(value)}</span>;
```

Comment:
This looks the same as the component for spec above. Should we reuse one component?